catalogue/views.py

```diff
@@ -143,12 +143,14 @@ def post(self, request, pk):
         return self.get(request)
 
     def delete(self, request, pk):
+        print(int(request.GET.get('c', -1)))
         request.session['cart'].remove(pk)
         return self.get(request)
 
     def get(self, request):
         if 'cart' not in request.session:
             request.session['cart'] = []
-        c_pk = request.GET.get('c', -1) # for detail view add button rendering
+        c_pk = int(request.GET.get('c', -1)) # current component
+        in_cart = True if c_pk in request.session['cart'] or c_pk == -1 else False
```

Comment:
`in_cart = c_pk in request.session['cart'] or c_pk == -1`

`TRUE if something else FALSE` is redundant